### PR TITLE
MM Deusillus, Ultra Graviton, Niffle Kong and It´s Coming

### PR DESCRIPTION
**Pull request by romanogit**
image

image

image

(Ultra Graviton is pending to resolve the bonus icons)

image

### CODE REVIEW

File: server/game/CompositeCard.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    moveTo(targetLocation) {
+        super.moveTo(targetLocation);
+
+        if(this.location === 'play area') {
+            this.compositeParts.forEach(id => {
+                // Always remove the second part from hand
+                let part = this.controller.getSourceList('hand').find(card => id === card.id);
+                this.controller.removeCardFromPile(part);
+                this.playedParts.push(part);
+            });
```
Comment (jeremylarner) [inline server/game/CompositeCard.js:22]: I think this code should be in PutIntoPlayAction

File: server/game/CompositeCard.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+            this.image = this.compositeImageId || this.id;
+        } else {
+            let cardPile = this.controller.getSourceList(this.location);
+            let cardIndex = cardPile.indexOf(this);
+            this.playedParts.forEach(part => {
+                part.location = this.location;
+                cardPile.splice(cardIndex, 0, part);
```
Comment (jeremylarner) [inline server/game/CompositeCard.js:30]: Have you tested this?  As far as I can see, the card isn't in a pile when moveTo is called.
I can't see how you're removing the composite card from the pile either....

Comment (romanogit) [reply server/game/CompositeCard.js]: Put this code in player

**Comment by romanogit** [comment]
See #1361

**Pull request closed without merging**